src/Models.jl

```diff
@@ -44,6 +45,26 @@ Returns a predictive distribution or point estimates depending on the [`Model`](
 """
 function predict end
 
+"""
+    submodels(model|template)
+
+Returns all submodels within a multistage model/template.
+Submodels are models within a model that have their own inputs (which may or may not be
+combined whith outputs of earlier models, before actually inputting them).
+Such multistage models take a tuple of inputs (which may be nested if the submodel itself
+has submodels).
+The order of inner models return by `submodels` is as per the order of the inputs in the
+tuple.
+
+For single-stage models, (i.e. ones that simply take a matrix as input), this returns a
+empty tuple.
+Wrapper models which do not expose their inner models to seperate inputs, including ones
```

Comment:
i'd be tempted to make this a note
```
!!! note 
    Wrapper models which ...
```

Reply:
I am not.
Its not a side point or a remark not to be missed, it is a core part of the point

Reply:
yeah, that's exactly why i think it should be in an admonition

Reply:
I am fine for now, but may revisit this when we fully formalize the API for multistage models